dpcpp/test/solver/idr_kernels.cpp

```diff
@@ -258,8 +258,8 @@ TEST_F(Idr, IdrStep3IsEquivalentToRef)
     GKO_ASSERT_MTX_NEAR(v, d_v, 2 * rr<value_type>::value);
     GKO_ASSERT_MTX_NEAR(u, d_u, 2 * rr<value_type>::value);
     GKO_ASSERT_MTX_NEAR(m, d_m, 2 * rr<value_type>::value);
-    GKO_ASSERT_MTX_NEAR(f, d_f, 13 * rr<value_type>::value);
-    GKO_ASSERT_MTX_NEAR(r, d_r, 2 * rr<value_type>::value);
+    GKO_ASSERT_MTX_NEAR(f, d_f, 100 * rr<value_type>::value);
```

Comment:
Apparently, a factor of 100 is still not enough:
See the [Idr.IdrStep3IsEquivalentToRef test for dpcpp/igpu/release/shared/dpcpp/test/solver/idr_kernels](https://gitlab.com/ginkgo-project/ginkgo-public-ci/-/jobs/2006877212).